showCompetences.js

```diff
@@ -0,0 +1,56 @@
+let users = [
+    {
+        userName: "Mario Rossi",
+        competenceUser: [
+            {
+                nameComp: 'java',
+                area: 'linguaggi programmazione',
+                level: 3
+            },
+            {
+                nameComp: 'Linux',
+                area: 'Sistemi operativi',
+                level: 2
+            }
+        ]
+    },
+    {
+        userName: "Franco Verdi",
+        competenceUser: [
+            {
+                nameComp: 'js',
+                area: 'linguaggi programmazione',
+                level: 4
+            },
+            {
+                nameComp: 'Windows',
+                area: 'Sistemi operativi',
+                level: 1
+            }
+        ]
+    }
+];
+
+for (let index = 0; index < users.length; index++) {
```

Comment:
Usa il "for of" tanto l'indice non lo usi:

for (const user of users) {
  // Something
}